CONTRIBUTING.rst

```diff
@@ -389,6 +392,21 @@ This is also useful for writing unit tests.
 Writing small functions is not always possible, and we do not recommend trying to reorganize larger,
 but well-tested, older functions in the codebase, unless there is a strong reason to do so (e.g., when adding a new feature).
 
+APIs of nilearn objects
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Estimated Attributes
+""""""""""""""""""""
+
+Attributes that have been estimated from the data
+must always have a name ending with trailing underscore.
+For example the coefficients of some regression estimator
+would be stored in a ``coef_`` attribute after ``fit`` has been called.
```

Comment:
Note that I reused the phrasing from scikit learn but I have not checked if it IS actually the case that all estimated attributes in our code base end with an underscore.

So I will turn the "must" in the sentence above to a "should" until we got that checked.

Could then open an issue to make sure we don't forget to check (and fix) any departure from this rule.